ang/crmCaseType.js

```diff
@@ -230,6 +249,12 @@
       }
     };
 
+    /// Clears the activity's default assignee values for relationship and contact
+    $scope.clearActivityDefaultAssigneeValues = function(activity) {
```

Comment:
`function (activity)` - space is needed.
What do you think, would running a manual semistandard help here?